### DISCUSSION TITLE
Automazione valvole riscaldamento a pavimento

### DISCUSSION THREAD

**Discussion by Mekki86** [indomus/forum#2645] [GENERALE - Discussioni generiche]
Ciao a tutti! Sono alle prime armi con la domotica. Avrei bisogno di automatizzare le valvole di una parte (per ora) del riscaldamento a pavimento. Sono 5 zone. Ho visto che ci sono parecchi attuatori che si possono montare sopra le valvole, ma sono da collegare al termostato. Io volevo semplicemente poterle comandare a distanza decidendo quando aprirle e quando chiuderle. Secondo voi si può fare? Magari comandandole con un sonoff o qualcosa di simile? 
Avrei poi intenzione di automatizzarle con HA attraverso alcuni sensori che ho già sparsi per casa. 
Grazie mille a chi avrà la pazienza di spiegarmi!

**Comment by dante70** [comment]
ciao,
il termine ''si devono collegare al termostato ''per agli attuatori si riferisce forse al collegamento che si usa piu' comunemente.
Tu se monti attuatori puoi alimentarli e/o comandarli come vuoi.
Come giustamente hai percepito se ad esempio ognuno di questi li colleghi ad un rele' di un son off ti viene esposto su homeassistant lo switch corrispondente.
una volta che hai quello puoi farci cio ' che vuoi e quindi automatizzare a piacimento.

**Comment by Mekki86** [comment]
Grazie mille!!
Per quanto riguarda il voltaggio devo prenderli a 230 V o 24 V?
A 2 Fili vanno bene?
Invece per quanto riguarda il sonoff deve essere a "contatto pulito" oppure no?
Sono proprio ignorante in materia!

**Reply by dante70** [reply]
per la scelta degli attuatori dipende da come vuoi fare l'impianto .
non conosco attuatori a 2 fili se non con ritorno a molla ,cioe'con una posizione di riposo che sia determinata da un ritorno meccanico in assenza di segnale.
non sono esperto di testine per impianti a pavimento ma altri attuatori a livello industriale li conosco.
prova a postare marca della miscelatrice e poi magari do una occhiata e ti dico la mia

**Comment by Mekki86** [comment]
La marca è Giacomini. Le valvole sono come quelle in questo kit: R553FK105!
Grazie

**Reply by dante70** [reply]
la testina giusta e' la R743 che puo' esserea 24 o 230 V
se le compri a 230 V e le colleghi ad esempio ad un son off puoi evitare altri rele' e accessori in quanto sono a 2 fili.
normalmente senza alimentazione sono chiuse ,se le alimenti aprono il passaggio per poi richiudersi se manca alimentazione in quanto il principio di funzionamento e' basato sulla dilatazione di un soffietto contenente cera che si dilata se scaldata.
per l'acquisto verifica bene con un rivenditore codici ecc.Ti serve anche secondo me chi ti fa collegamenti a regola d'arte e quindi un elettricista

**Comment by Mekki86** [comment]
Ok, grazie mille davvero. 
Il fatto che uno dei quattro attuatori molto probabilmente rimarrà aperto tutta la stagione, influisce su qualcosa? A livello di consumi cambia qualcosa tra 230 e 24V? Quello che mi chiedo è: a tenere aperta la valvola, l'attuatore consuma o chiede corrente solo per aprirlo?

**Reply by dante70** [reply]
la potenza e' la stessa ovviamente a 24 o 230V.
Per il consumo io non mi preoccuperei .....consuma 2.5 Watt   se ti fai il conto della serva come si suol dire 2.5x24x180=11KWH per 6 mesi
guardati una bolletta dell'energia elettrica e vedi cosa paghi un KWH     e' un costo irrisorio se consideriamo tutti i consumi che abbiamo comunque sempre nelle nostre abitazioni tanto piu alti di questo.

**Comment by Mekki86** [comment]
Ok, ultima domanda, meglio in ch4 pro o un ch4 normale per controllarle?

**Reply by dante70** [reply]
scegli tu in base a come farai imianto.
Se usi attautori a 230 V non ti serve il contatto pulito che ha la versione pro

**Comment by Mekki86** [comment]
Ok, grazie ancora. Ti farò sapere!
